Library/Homebrew/cask/config.rb

```diff
@@ -16,19 +16,19 @@ class Config
       {
         appdir:               "/Applications",
         keyboard_layoutdir:   "/Library/Keyboard Layouts",
-        colorpickerdir:       "~/Library/ColorPickers",
-        prefpanedir:          "~/Library/PreferencePanes",
-        qlplugindir:          "~/Library/QuickLook",
-        mdimporterdir:        "~/Library/Spotlight",
-        dictionarydir:        "~/Library/Dictionaries",
-        fontdir:              "~/Library/Fonts",
-        servicedir:           "~/Library/Services",
-        input_methoddir:      "~/Library/Input Methods",
-        internet_plugindir:   "~/Library/Internet Plug-Ins",
-        audio_unit_plugindir: "~/Library/Audio/Plug-Ins/Components",
-        vst_plugindir:        "~/Library/Audio/Plug-Ins/VST",
-        vst3_plugindir:       "~/Library/Audio/Plug-Ins/VST3",
-        screen_saverdir:      "~/Library/Screen Savers",
+        colorpickerdir:       "${HOME}/Library/ColorPickers",
```

Comment:
Does `Pathname("${HOME}").expand_path` work?

Reply:
@Bo98 No. I'll revert the `$HOME` changes.